Tasks/VsTest/testselectorinvoker.ts

```diff
@@ -106,6 +107,109 @@ export class TestSelectorInvoker {
         return output.code;
     }
 
+    public publishCodeChangesInDistributedMode(inputDataContract: inputdatacontract.InputDataContract): number {
```

Comment:
Help me understand this. Why did we create a new invoker for distributed?

Reply:
because inputs need to come from inputdatacontract which currently does not exist for single agent flow.